newIDE/app/src/InstancesEditor/InstancesRotator.js

```diff
@@ -0,0 +1,40 @@
+export default class InstancesRotator {
+  constructor() {
+    this.instanceAngles = {};
+    this.totalDeltaX = 0;
+    this.totalDeltaY = 0;
+  }
+
+  setOptions(options) {
+    this.options = options;
```

Comment:
@blurymind Can you remove this too? :)

Reply:
oops, I forgot to remove it in the last comit
Removed now :)